app/src/services/sync.js

```diff
@@ -329,15 +329,18 @@ const service = {
       // S3 Tags
       const s3Tags = toLowerKeys(s3TagsForVersion?.TagSet ?? []);
 
-      // Ensure `coms-id` tag exists on this version in S3
-      if (s3Tags.length < 10 && !s3Tags.find(s3T => s3T.key === 'coms-id')) {
+      // Add coms-id tag to latest version in S3 if not already present
+      // note: For a sync job the _deriveObjectId() function will have already added
+      // the coms-id to latest version.
+      if (version.isLatest && s3Tags.length < 10 && !s3Tags.find(s3T => s3T.key === 'coms-id')) {
+        // note: add tags to a specified version (passing a `VersionId` parameter) will affect `Last Modified`
+        // object attribute of versions on some s3 storage providors including Dell ECS
         await storageService.putObjectTagging({
           filePath: path,
           tags: (s3TagsForVersion?.TagSet ?? []).concat([{
             Key: 'coms-id',
             Value: comsVersion.objectId
           }]),
-          s3VersionId: comsVersion.s3VersionId,
```

Comment:
May be worth keeping this line in as a commented out value with the above note instead of outright removing it.